packages/gafl-webapp-service/src/pages/contact/name/__tests__/route.spec.js

```diff
@@ -14,16 +17,22 @@ describe('name > route', () => {
   }
 
   describe('getData', () => {
-    it('should isLicenceForYou as true if isLicenceForYou is true on the status cache', async () => {
+    it('should return isLicenceForYou as true if isLicenceForYou is true on the status cache', async () => {
       mockStatusCacheGet.mockImplementationOnce(() => ({ isLicenceForYou: true }))
       const result = await getData(mockRequest)
       expect(result.isLicenceForYou).toBeTruthy()
     })
 
-    it('should isLicenceForYou as false if isLicenceForYou is false on the status false', async () => {
+    it('should return isLicenceForYou as false if isLicenceForYou is false on the status cache', async () => {
       mockStatusCacheGet.mockImplementationOnce(() => ({ isLicenceForYou: false }))
       const result = await getData(mockRequest)
       expect(result.isLicenceForYou).toBeFalsy()
     })
+
+    it('should call getPronoun', async () => {
+      mockStatusCacheGet.mockImplementationOnce(() => ({ isLicenceForYou: true }))
+      await getData(mockRequest)
+      expect(getPronoun).toHaveBeenCalled()
+    })
```

Comment:
this test isn't quite right. We need to test the new return value from the getData method (pronoun) is what we'd expect given different scenarios. Whether it calls getPronoun should be incidental - it doesn't matter _how_ it produces its outputs, just that it produces the correct ones.